docs/conceptual/node/tx-pool.md

```diff
@@ -2,12 +2,80 @@
 title: Transaction Pool
 ---
 
-* What is the Transaction Pool?
 
-* What is priority?
+The transaction pool contains all transactions (signed and unsigned) broadcasted to the network that have been *received* and *validated* by the local node. The pool is responsible for:
 
-* What is dependency?
 
-* Graph VS Queue
+1) Checking if transactions are valid. `validity` of the transaction is not hard wired to the transaction pool, but is defined by the runtime through the `validate_transaction` function ([code](https://github.com/paritytech/substrate/blob/20a9b15cdbed4bf962a4447e8bfb812f766f2fbc/frame/executive/src/lib.rs#L304)). Example validity checks are:
 
-* What is the lifecycle of a transaction?
+* Transaction Index (nonce) correctness.
+* The account has enough funds to pay for the associated fees.
+* The signature is valid.
+
+2) If the transaction is valid, sorting the transactions into 2 transaction queues:
+   - **Ready Queue** -  Contains transactions that can be included in a new pending block and must follow the *exact* order in the ready queue.
+
+   - **Future Queue** - Contains transactions that *seem* valid so far, but the full node has yet to see the remaining transactions it depends on. For example, *Transaction B* is in the future queue and can only be included in a block after *Transaction A* has been included. However, the node is yet to see *Transaction A* in its transaction pool.
+
+Full nodes in the network all have their own local transaction pool and transaction queues. Therefore, full nodes may have a different view of the queues and transactions in the network.
+
+Dependent on the runtime, strict ordering may not be a requirement. This could enable full nodes to implement different strategies on how they prioritize propagating transactions and how they're included into blocks.
+
+## Transaction Dependencies
+
+Transaction dependencies are defined by the `requires` and `provides` parameters in the `ValidTransaction` [struct](https://substrate.dev/rustdocs/master/sp_runtime/transaction_validity/struct.ValidTransaction.html) to build a dependency graph of transactions. Together with `priority`, this allows the pool to traverse the graph and produce a valid linear ordering of transactions to be propagated and included in the next block.
+
+For `Frame` based runtimes, an account based system is used for transaction ordering. Every signed transaction needs to contain a nonce, which is incremented by 1 every time a new transaction is made. For example, the first transaction from a new account will have `nonce=0` and the second transaction will have `nonce=1`.
+
+At minimum, `Frame` transactions have a `provide` tag of `encode(sender ++ nonce)` and a `require` tag of `encode(sender ++ (nonce -1)) if nonce > 1`. Transactions do not `require` anything if `nonce=0`. As a result, all transactions coming from a single sender will form a sequence in which they should be included.
```

Comment:
In the first sentence you define `require` as a noun but in the second you use it as a verb. I think in the second sentence it should not be in pre-formatted text.